src/components/CallView/shared/Video.vue

```diff
@@ -180,6 +185,41 @@ export default {
 			return this.isNotConnected && this.model.attributes.connectionState !== ConnectionState.FAILED_NO_RESTART
 		},
 
+		isDisconnected() {
+			return this.model.attributes.connectionState !== ConnectionState.NEW && this.model.attributes.connectionState !== ConnectionState.CHECKING
+				&& this.model.attributes.connectionState !== ConnectionState.CONNECTED && this.model.attributes.connectionState !== ConnectionState.COMPLETED
+		},
+
+		/**
+		 * Whether the connection to the participant is being tried again.
+		 *
+		 * The initial connection to the participant is excluded.
+		 *
+		 * A "failed" connection state will trigger a reconnection, but that may
+		 * not immediately change the "negotiating" or "connecting" attributes
+		 * (for example, while the new offer requested to the HPB was not
+		 * received yet). Similarly both "negotiating" and "connecting" need to
+		 * be checked, as the negotiation will start before the connection
+		 * attempt is started.
+		 */
+		isReconnecting() {
+			return this.model.attributes.connectionState === ConnectionState.FAILED
+				|| (!this.model.attributes.initialConnection
+					&& (this.model.attributes.negotiating || this.model.attributes.connecting))
+		},
+
+		connectionMessage() {
+			if (this.isReconnecting) {
+				return t('spreed', 'Connection problems. Trying to reconnect …')
```

Comment:
```suggestion
				return t('spreed', 'Trying to reconnect …')
```
Would even do only this

Reply:
Originally I did that, but later I changed it to also include the _Connection problems_ part. The thing is that I do not remember why... :facepalm: So fine by me to shorten it again I guess :-P

Reply:
I would say 
"Connection lost, trying to reconnect"